src/scss/base.scss

```diff
@@ -29,6 +34,65 @@ body {
   display: flex;
 }
 
+.features-container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+}
+
+.features-decoration {
+  background: url(../assets/triangle.png);
+  width: 100%;
+  height: 10px;
+}
+
+.features-container {
+  background: $light-grey;
+}
+
+.features-wrapper {
+  background: $light-grey;
+  width: 100%;
+  max-width: 1280px;
+  padding: 50px;
+  display: flex;
+  flex-direction: row;
+  align-self: center;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.feature-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 200px;
+  justify-content: center;
+  align-items: center;
+}
+
+.feature-title {
+  display: flex;
+  font-weight: bold;
+  font-size: 11pt;
+  color: $dark-grey;
+  margin: 0 0 15px 0;
```

Comment:
the reason for overriding all this margins (also the one added in the body) is because you are not using a css reset file, use the equalizer for this. you can find it in the tech-guides repo under frontend/layout in open source.

to learn more about css resets read this:

http://cssreset.com/what-is-a-css-reset/